lighthouse-cli/test/smokehouse/test-definitions/legacy-javascript/expectations.js

```diff
@@ -21,6 +21,56 @@ module.exports = [
                 url: 'http://localhost:10200/legacy-javascript.js',
                 subItems: {
                   items: [
+                    {signal: 'Array.prototype.fill'},
+                    {signal: 'Array.prototype.filter'},
+                    {signal: 'Array.prototype.findIndex'},
+                    {signal: 'Array.prototype.find'},
+                    {signal: 'Array.prototype.forEach'},
+                    {signal: 'Array.from'},
+                    {signal: 'Array.prototype.includes'},
+                    {signal: 'Array.isArray'},
+                    {signal: 'Array.prototype.map'},
+                    {signal: 'Array.of'},
+                    {signal: 'Array.prototype.reduceRight'},
+                    {signal: 'Array.prototype.reduce'},
+                    {signal: 'Array.prototype.some'},
+                    {signal: 'Date.now'},
+                    {signal: 'Date.prototype.toISOString'},
+                    {signal: 'Date.prototype.toJSON'},
+                    {signal: 'Number.isInteger'},
+                    {signal: 'Number.isSafeInteger'},
+                    {signal: 'Number.parseInt'},
+                    {signal: 'Object.defineProperties'},
+                    {signal: 'Object.defineProperty'},
+                    {signal: 'Object.entries'},
+                    {signal: 'Object.freeze'},
+                    {signal: 'Object.getOwnPropertyDescriptors'},
+                    {signal: 'Object.getOwnPropertyNames'},
+                    {signal: 'Object.getPrototypeOf'},
+                    {signal: 'Object.isExtensible'},
+                    {signal: 'Object.isFrozen'},
+                    {signal: 'Object.isSealed'},
+                    {signal: 'Object.keys'},
+                    {signal: 'Object.preventExtensions'},
+                    {signal: 'Object.seal'},
+                    {signal: 'Object.setPrototypeOf'},
+                    {signal: 'Object.values'},
+                    {signal: 'Reflect.apply'},
+                    {signal: 'Reflect.construct'},
+                    {signal: 'Reflect.defineProperty'},
+                    {signal: 'Reflect.deleteProperty'},
+                    {signal: 'Reflect.getOwnPropertyDescriptor'},
+                    {signal: 'Reflect.getPrototypeOf'},
+                    {signal: 'Reflect.get'},
+                    {signal: 'Reflect.has'},
+                    {signal: 'Reflect.isExtensible'},
+                    {signal: 'Reflect.ownKeys'},
+                    {signal: 'Reflect.preventExtensions'},
+                    {signal: 'Reflect.setPrototypeOf'},
+                    {signal: 'String.prototype.codePointAt'},
+                    {signal: 'String.fromCodePoint'},
+                    {signal: 'String.raw'},
+                    {signal: 'String.prototype.repeat'},
```

Comment:
This should have been a clue something was wrong. the `legacy-javascript.js` fixture here is the `core-js@3 esmodules: false` variant, which should have dozens of variants.